tools/git/hooks/src/pre_commit.py

```diff
@@ -35,14 +39,17 @@ def parse_args():
 
     parser.add_argument('--uncrustify', default = False, action='store_true', help='Uncrustify modified files.')
     parser.add_argument('--uncrustify-all-files', default = False, action='store_true', help='Uncrustify all files.')
+    parser.add_argument('files', nargs='*', help='List of files to check.')
 
     args = parser.parse_args()
     return vars(args)
 
 
-def commit_is_ready():
+def commit_is_ready(file_names=[]):
```

Comment:
Default argument in python are static variable to the function (one of the gotchas), so unless you want to maintain a state of the list, use `files_names=None` here.

Reply:
Done.